deployment/terraform/resources/functions.tf

```diff
@@ -46,7 +46,7 @@ resource "azurerm_function_app" "pcfuncs" {
   os_type = "linux"
   version = "~4"
   site_config {
-    linux_fx_version          = "PYTHON|3.8"
+    linux_fx_version          = "PYTHON|3.9"
```

Comment:
Unrelated, but updating to use the same version that pccommon (which is used in pcfunc) also executes on.